### ISSUE TITLE
Create new SD Table for automodes

### ISSUE THREAD

**Issue by Twinters007** [robotpy/robotpy-wpilib-utilities#48]
Currently, autonomous variables add variables to the path SmartDashboard/MODE/VAR, but they are all under the SmartDashboard table. I propose creating a new table for each Automode, so they show up under their own folder. Much cleaner that way

**Comment by Twinters007** [comment]
...this isn't a thing

**Comment by Twinters007** [comment]
It may also be good to remove the mode name if it's in the MODE_NAME table

**Comment by virtuald** [comment]
Yeah, I had meant to change how that worked, but it's too late for 2017 (don't want to break existing code).

**Comment by Twinters007** [comment]
i decided that too
